chapters/6-analysis.tex

```diff
@@ -147,13 +147,19 @@ \subsection{Fitting construction into the data workflow}
 in a unified script, separate from the analysis code, helps
 avoid this and ensures consistency across different outputs.
 
-\subsection{Integrating different data sources}
+\subsection{Combining different data sources}
 
 % When merging is necessary and how to start thinking about it
 To create the analysis dataset,
-it is typically necessary to combine or merge information
-from different data sources.\sidenote{
-	\url{https://dimewiki.worldbank.org/Data_Integration}}
+it is typically necessary to combine information
+from different data sources.
```

Comment:
here is probably where we want to link to the data integration page on the dime wiki (currently just a stub)

Reply:
done in 484ae23d18a7f2f5638b1a25d46fb3fcab39cd12